_toc.yml

```diff
@@ -1,42 +1,62 @@
 - file: index
 - chapters:
-  - file: community/index
-    sections:
-    - file: community/mission_and_values
-    - file: community/team
-    - file: community/code_of_conduct
-    - file: community/code_of_conduct_reporting
-    - file: community/governance
-    title: Community
+  - file: tutorials/start_index
+    sections: 
+    - file: tutorials/fundamentals/installation
+    - file: tutorials/absolute_beginners
+    - file: tutorials/fundamentals/getting_started
+    - file: plugins/stable/find-and-install-plugin 
+    - file: tutorials/fundamentals/viewer
+    title: Getting started
   - file: tutorials/index
     sections:
-    - file: tutorials/fundamentals/index
-      sections:
-      - file: tutorials/fundamentals/installation
-      - file: tutorials/fundamentals/getting_started
-      - file: tutorials/fundamentals/viewer
-      - file: tutorials/fundamentals/image
-      - file: tutorials/fundamentals/labels
-      - file: tutorials/fundamentals/points
-      - file: tutorials/fundamentals/shapes
-      - file: tutorials/fundamentals/surface
-      - file: tutorials/fundamentals/tracks
-      - file: tutorials/fundamentals/vectors
-    - file: tutorials/applications/index
-      sections:
-      - file: tutorials/applications/annotate_segmentation
-      - file: tutorials/applications/annotate_points
-      - file: tutorials/applications/dask
-      - file: tutorials/applications/napari_imageJ
-      - file: tutorials/applications/cell_tracking
+    #- file: tutorials/fundamentals/index
+    #- file: tutorials/applications/index
+    - file: tutorials/applications/annotate_segmentation
+    - file: tutorials/applications/annotate_points
+    - file: tutorials/applications/dask
+    - file: tutorials/applications/cell_tracking
```

Comment:
Suggestion: have separate subsections for "Segmentation" and "Tracking". Those are very common workflows and will likely hold many pages on different examples in the future.

Reply:
If we do create these two subsections, do you propose we move the existing documents under there? For example:

**Tutorials:**
- Segmentation
   - Annotate segmentation
   - ...
- Tracking
   -  Cell tracking
   - ...
- Annotating videos with napari
- Using dask and napari
- Tracks layer tutorial

If we do this, note that the third level (documents inside the Segmentation section and Tracking section) will not appear in the main ToC. Is that what we want?